File: 프로그래머스/unrated/181900. 글자 지우기/글자 지우기.js
function solution(my_string, indices) {
    indices.sort((a, b) => a - b);
    
    let answer = '';
    
    for (let i = 0; i < my_string.length; i++) {
        if (indices.includes(i)) {
            continue;
        }
        
        answer += my_string[i];
    }
    
    return answer;
}

File: 프로그래머스/unrated/181900. 글자 지우기/글자 지우기.java
import java.util.Hashtable;

class Solution {
    public String solution(String my_string, int[] indices) {
        String answer = "";
        
        Hashtable<Integer, Integer> hash = new Hashtable<Integer, Integer>(indices.length);
        for (int index : indices) {
            hash.put(index, index);
        }
        
        for (int i = 0; i < my_string.length(); i++) {
            if (hash.containsKey(i)) {
                continue;
            }
            
            answer += Character.toString(my_string.charAt(i));
        }
        
        return answer;
    }
}